src/components/WalletUnlock/DeterministicAccountTable.tsx

```diff
@@ -0,0 +1,334 @@
+import React, { useContext } from 'react';
+import styled, { css } from 'styled-components';
+import { DWAccountDisplay, AddressBookContext, fromTokenBase } from '@services';
+import { COLORS, SPACING, BREAK_POINTS } from '@theme';
+import translate, { Trans } from '@translations';
+import { Identicon } from '@mycrypto/ui';
+import {
+  EditableAccountLabel,
+  EthAddress,
+  Typography,
+  LinkOut,
+  Button,
+  Tooltip
+} from '@components';
+import { Network, ExtendedAsset, TAddress } from '@types';
+import { isSameAddress } from '@utils';
+import BN from 'bn.js';
```

Comment:
Prefer `bigify`

Reply:
Import order

Reply:
We can't since `formTokenBase` use `TokenValue` type which is a `BN` and `bigify` returns a `BigNumber`, it will need too much change for this PR.